src/app/models/book-model.ts

```diff
@@ -0,0 +1,28 @@
+import { bookType } from '../book-type.enum';
+import { BookBorrowData } from './book-borrow-data-model';
+
+export class Book {
+  public id: number;
+  public nameBook: string;
```

Comment:
Just 'name' would be better property name. Calling this prop would be probably something like: book.name